server/routes/auth/auth.js

```diff
@@ -2,6 +2,8 @@ const express = require('express');
 const router  = express.Router();
 const passport = require('passport');
 const db = require('../../db');
+require('dotenv').config();
```

Comment:
Redundant, you only need to conigure once, and we did it in index.js

Reply:
OK good to know, thanks, I removed it